src/index.js

```diff
@@ -335,14 +335,10 @@ function ncc (
               compiler: eval('__dirname + "/typescript.js"'),
               compilerOptions: {
                 allowSyntheticDefaultImports: true,
-                module: 'esnext',
-                outDir: '//',
-                ...(fullTsconfig &&
-                  fullTsconfig.compilerOptions &&
-                  fullTsconfig.compilerOptions.incremental
-                    ? { incremental: false }
-                    : {}),
```

Comment:
Wait I only just saw this isn't using the TSConfig at all. Shouldn't this be `...fullTsconfig.compilerOptions` for all compiler options?

Reply:
Yeah good point, we'll see if all the tests pass or if there are other options we need to override